app/main/views/users.py

```diff
@@ -36,6 +37,15 @@ def get_user_by_id(user_id):
     return jsonify(users=user.serialize())
 
 
+@main.route('/users', methods=['GET'])
+def get_user_by_email():
+    email_address = request.args.get('email')
```

Comment:
This effectively a required field. I think it would be useful to give a useful 400 error message if it's not provided.